doc/tutorials/10-reaction_ensemble/10-reaction_ensemble.tex

```diff
@@ -315,6 +321,8 @@ \subsection{System setup}
 reactant_coefficients=[1,1], product_types=[],    product_coefficients=[],    
 default_charges={type_H: charges[type_H], type_OH: charges[type_OH]})
 \end{verbatim}
+Please notice that the order in which the species are written in reactants and products lists is very important because, when a reaction is performed, the first species in the reactants list is replaced by the first species in the product lists, the second reactant species is replaced by the second product species, and so on. Moreover, if the reactant list has more species than the products list, reactant molecules in excess are deleted from the system, while if the products list have more species than the reactants list, products molecules in excess are created and randomly placed inside the simulation box. For example, reversing the order of products in our reaction (i.e. from  \verb|product_types=[type_H, type_A]| to \verb|product_types=[type_A, type_H]|), a neutral monomer would be positively charged and a negative monovalent counterion would be randomly placed inside the cell. 
```

Comment:
[...] while if the products list **has** more species than the reactants list, **product** molecules [...]